src/packages/@library/ui/button/Button.svelte

```diff
@@ -135,4 +144,13 @@
   }
   .custom-btn-transparent:hover {
   }
+  .custom-btn-teritiary{
+    color: var(--text-primary-300); 
+    border: 1px solid var(--border-primary-200) !important;
+  }
+  .custom-btn-teritiary:hover{
+    color: var(--white-color); 
+    background-color: var(--bg-primary-300 );
```

Comment:
color: var(--text

Reply:
class name tertiary that represents violet not matching with background property that uses color primary that represents blue

Reply:
okay looking into it.